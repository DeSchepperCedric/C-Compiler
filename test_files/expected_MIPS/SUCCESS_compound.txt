MARS 4.5  Copyright 2003-2014 Pete Sanderson and Kenneth Vollmar

i = 10. Expected: 10
i = 10. Expected: 10
i = 25. Expected: 25
i = 10. Expected: 10
