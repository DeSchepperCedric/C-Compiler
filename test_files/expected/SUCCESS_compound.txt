i = 10. Expected: 10
i = 10. Expected: 10
i = 25. Expected: 25
i = 10. Expected: 10
